ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.yaml",
```

Comment:
For compact cluster, we are setting `SIZE_VARIANT: "compact"` ENV in job configuration level, so for the workflow, you are supposed to create a non-compact workflow here, that means, the path should be like `gcp/ipi/filestore-csi/`

Reply:
so you decide not combine the filestore-csi to the compact install workflow?

Reply:
If set `SIZE_VARIANT: "compact"` workers will be 0 right? I prefer to
create master + worker env https://github.com/openshift/release/blob/1ed184c3693b8e2680ca0c6e0e94cd0e34d8fd23/ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh#L16

Reply:
> If set SIZE_VARIANT: "compact" workers will be 0 right? 

The cluster would only have 3 nodes, each node take both master and worker roles. I thought you select compact install workflow to combine filestore-csi on purpose. If you prefer a more common cluster, you can select other install workflow to combine your feature, e.g: gcp-ipi-ovn-ipsec-p1-f4